admin/templates/configuration/tmp/configure/sql/CssData.sql

```diff
@@ -0,0 +1,36 @@
+-- MySQL Script generated by MySQL Workbench
+-- Thu Jul 30 11:58:38 2015
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+-- -----------------------------------------------------
+-- Schema qservCssData
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema qservCssData
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `qservCssData` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
+USE `qservCssData`;
+
+-- -----------------------------------------------------
+-- Table `qservCssData`.`kvData`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `qservCssData`.`kvData` (
+  `kvKey` CHAR(255) NOT NULL COMMENT 'slash delimited path',
+  `kvVal` TEXT(1024) NOT NULL COMMENT 'value for key',
+  `parentKvId` INT UNSIGNED NULL COMMENT 'id of parent key',
+  `kvId` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  PRIMARY KEY (`kvId`),
+  UNIQUE INDEX `kvKey_UNIQUE` (`kvKey` ASC))
+ENGINE = InnoDB;
```

Comment:
Some improvements for this schema I think may be useful:
- I believe it's custom to define primary key column(s) at the begging, I'd move `kvId` to the top
- add foreign key constraint for `parentKvId` to make sure that data is consistent
- `TEXT(1024)` is the same storage type as `TEXT` and its size is limited to 64 kBytes, we probably want larger limit so I suggest to replace it with `MEDIUMTEXT`
- `UNSIGNED` is mysql-ism, we want to avoid it